src/app/components/Sorter/Sorter.jsx

```diff
@@ -61,31 +75,54 @@ class Sorter extends React.Component {
     this.setState({ active: false });
   }
 
-  handleOnClickOut() {
-    if (this.state.active) {
-      this.setState({ active: false });
-    }
+  /**
+   * renderResultsSort()
+   * The fuction that makes renders the sort options.
+   *
+   * @return {HTML Element}
+   */
+  renderResultsSort() {
+    return sortingOpts.map((d, i) => (
+      <option value={d.val} key={i}>
+        {d.label}
+      </option>
+    ));
   }
 
   render() {
+    const keywords = this.props.keywords || '';
+    const field = this.props.field || '';
+
     return (
       <div className="nypl-results-sorting-controls">
         <div className="nypl-results-sorter">
-          <ClickOutHandler onClickOut={() => this.handleOnClickOut()}>
-            <button
-              aria-expanded={this.state.active}
-              className={this.state.active ? 'active' : ''}
-              onClick={e => this.getResultsWindow(e)}
-            >
-              <span>Sort by <strong>{this.state.sortLabel}</strong></span>
-              <DownWedgeIcon className="nypl-icon" viewBox="0 0 68 24" />
-            </button>
-            <ul className={this.state.active ? '' : 'hidden'}>
-              {
-                this.getResultsSort()
-              }
-            </ul>
-          </ClickOutHandler>
+          <form
+            action={
+              `/search${keywords ? `?q=${keywords}` : ''}${field ? `&search_scope=${field}` : ''}`
```

Comment:
I noticed the case as below,
1) search `lemon` and after get the results, sort by `title (a-z)`
2) after get the sorted results, try to sort again, this time, sort by `date (new to old)`

When js is on, it will work perfectly, and we will get the same results with different sorting, but when js is off, the second step will return the corrected sorted results without search keywords `lemon`.

I guess it is because when no-js, Actions won't update the value of keywords in search component.

And the same reason as I included `search_scope` here for the case if the user has put in any facets in the future.